### PR TITLE
Some changes

### PR DESCRIPTION
**Pull request by carloscarcamo**
- Bug #20 fixed
- Allow to search multiple keywords, see #18

### CODE REVIEW

**Comment by anvk** [comment]
@carloscarcamo as I mentioned - I'm in a process of switching this project to React + ES6 + Webpack + Redux.
I pushed some of my work accomplished in React. Is it possible for you to update your pull request accordingly? It should be fairly easy. UI components you are looking for will be located in `client/src/components/mapPage`

Also how familiar you are with React ?

File: server/routes/search.js [hunk 1]
```diff
@@ -6,34 +8,50 @@ var searchRoutes = function(router) {
...
+        queries = [];
 
-    //execute db query
-    collection.find({
-      $text: {
-        $search: req.body.tweetText
-      }
-    }, { tln: 1, tlt: 1 }, function(err, cursor) {
+    // multiple find statements for search multiple keywords
+    q.forEach(function(v, i, a){
```
Comment (anvk) [inline server/routes/search.js]: I probably should eventually find time and create proper eslint file. But could you keep a space between `) {`

**Comment by carloscarcamo** [comment]
Ok, I will send another pull request, this time I will fork the project again into our Mozilla community github page :)

I'm not familiar with React but, I think this project could be a good starting point :)

**Comment by anvk** [comment]
@carloscarcamo if you want you can wait a bit because I'm going to add Webpack and ES6 support to the project in the next few days.

**Comment by carloscarcamo** [comment]
Ok. I already did some changes but, I will wait for yours :)

**Comment by anvk** [comment]
@carloscarcamo actually it is great that you are making changes for this project. Let's do otherwise - I will wait for your changes and we will merge them in and then I will move project from ES5 to ES6 JavaScript

**Pull request closed without merging**